networkbuildercoordinator.py

```diff
@@ -8,7 +8,7 @@
 import time
 
 
-def main(output_path=None):
+def main(output_path=None,neodb=False):
```

Comment:
Instead of adding this new variable here (and the corresponding check further down in the script), it'd be cleaner if there's a separate script that reads the network in pkl from a given directory and transforms it to neo4. There are 2 advantages of that approach: i) we do not need to include a variable for each new backend here; ii) each new backend has its own self-contained script.